pyproject.toml

```diff
@@ -239,7 +231,7 @@ extend-select = [
   "FIX", # flake8-fixme
   "FLY", # flynt
   "G", # flake8-logging-format
-  "I002", # missing-required-import
+  "I", # isort
   "ICN", # flake8-import-conventions
```

Comment:
This is us telling it to use isort instead of ruff, right?

Reply:
Correct.  The `I` rule set for ruff contains `I001` (which sorts imports) and `I002` (which checks for missing imports). We didn't enable `I001` earlier because `isort` was doing the import sorting for us.